### PR TITLE
Update README and CONTRIBUTING docs

### PR DESCRIPTION
**Pull request by githubstudent**
Small grammar, spelling, and formatting updates to all the docs for this project.

Looking for some review and :eyes: on this and a :+1: from a colleague before we merge and :ship:.

### CODE REVIEW

**Comment by githubstudent** [comment]
@githubschool/developers, Anyone have a chance to review this for me? Thanks!

**Comment by githubteacher** [comment]
Looking good but the `contributing.md` needs a bit more polish. Commit those changes and I'll be happy to review again. 

If I have spare cycles, I'll make a few changes myself to help out.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
 # Example Repo for Foundations Class
-## January 7, 2015
 
+**Event Date:** January 7, 2015
+
+**Teacher:** Jordan McCullough
+
+## Description
```
Comment (githubteacher) [inline README.md]: @githubstudent, Do you mind that I added this `h2

**Comment by githubstudent** [comment]
This is looking fine to me. Let's :ship: this when you have a chance to review and `:+1:` this.

**Comment by githubteacher** [comment]
Did we need to cross-reference any Issues that were related to this? Reviewing the Issues now.

**Pull request merged**